utils/guidelines_xslt/odd2html/generateWebsite.xsl

```diff
@@ -91,11 +93,15 @@
         <xsl:variable name="quot" as="xs:string">"</xsl:variable>
         <xsl:variable name="json.string" select="replace(replace(replace(xml-to-json($search.index),'\\/','/'),'\{', $newline || '{'),'(' || $quot || '(text|title|url|tags)' || $quot || ')',$newline || '  $1')" as="xs:string"/>
         
-        <xsl:result-document href="{$web.output}tipuesearch/search_content.js" method="text" indent="yes" omit-xml-declaration="yes" >
+        <!--<xsl:result-document href="{$web.output}tipuesearch/search_content.js" method="text" indent="yes" omit-xml-declaration="yes" >
             var tipuesearch = <xsl:value-of select="$json.string"/>;
+        </xsl:result-document>-->
```

Comment:
if this is broken anyway, why not remove it completely?

Reply:
I haven't had the time to do a proper cleanup – that's one of the reasons why I said there's no rush to merge this.